app/services/reporter.rb

```diff
@@ -6,38 +6,76 @@
 # run queries and produce reports from the results, for consumption
 # by preservation catalog maintainers
 class Reporter
-  # @param [String] the name of the storage root for which druids should be listed
+  attr_reader :storage_root, :druids
+
+  # @param [String] the name of the storage root to initialize for reporter
```

Comment:
Should this method take `params` as a hash (in which case the YARD up here needs tweaking), or should it take the storage root name as a string param (in which case lines 13-14 need tweaking)?